afs-ext-base/src/test/java/com/powsybl/afs/ext/base/ImportedCaseTest.java

```diff
@@ -207,4 +208,25 @@ public void testNetwork() {
         assertNotNull(importedCase2);
         assertEquals("NetworkID", importedCase2.getName());
     }
+
+    @Test
+    public void testDataStore() throws IOException {
+        Folder root = afs.getRootFolder();
+
+        // create project
+        Project project = root.createProject("project");
+        assertNotNull(project);
+
+        // create project folder
+        ProjectFolder folder = project.getRootFolder().createFolder("folder");
+        assertTrue(folder.getChildren().isEmpty());
+
+        ImportedCase importedCase = folder.fileBuilder(ImportedCaseBuilder.class)
+                .withDatastore(DataStores.createDataStore(fileSystem.getPath("/work")))
+                .withName("network.tst")
+                .build();
+        assertNotNull(importedCase);
+        assertEquals("network.tst", importedCase.getName());
+        assertNotNull(importedCase.getNetwork());
```

Comment:
I think we should add additional test for the "reading" part, ie the `getDataStore` method.

We can check that it contains the single entry "network.tst".